src/components/Rockets.js

```diff
@@ -25,7 +25,7 @@ const Rocket = (props) => {
         <div>
           <span className={style.rocketName}>{name}</span>
         </div>
-        <div className={style.description} data-testid="reserved">
+        <div className={style.description}>
           {reserved && (
             <span className={style.badge}>Reserved</span>
           )}
```

Comment:
- [ ] Kindly add some styles to have space at the left side of these rocket images.

![image](https://user-images.githubusercontent.com/65860678/203827407-65628070-91f4-4d24-b98a-5c090d169c57.png)

Reply:
Will do, thanks!